utils/install_v2.sh

```diff
@@ -12,6 +12,27 @@ YELLOW=$'\e[0;33m'
 NC=$'\e[0m' # No Color
 TMP_DIR="/tmp/wasmedge.$$"
 
+# OS Detection Block
+# This block detects if the script is being run on an unsupported Windows environment.
+# "uname -s" outputs:
+# - 'Linux' on Linux
+# - 'Darwin' on macOS
+# - 'MINGW64_NT-xxxx' or similar on Windows Git Bash
+# - 'Windows_NT' in certain Windows environments
+#
+# If the OS is detected as Windows (Git Bash or native), the script exits early
+# and provides a link for manual installation
+
+OS=$(uname -s)
+
+if [[ "$OS" =~ ^MINGW.* || "$OS" = "Windows_NT" ]]; then
+    echo "Error: This script does not support Windows directly (including Git Bash)."
```

Comment:
Maybe add `This script does not support "$OS" ...` so that we know what `OS` was set to